home/jobs/OMERO-server/config.xml

```diff
@@ -121,7 +107,19 @@ unzip $ZIP
 rm -f $ZIP
 mv $WORKSPACE/$DIST $OMERO_DIST
 
+rm -rf $WORKSPACE/.venv3
+python3 -m venv $WORKSPACE/.venv3
+
 source $WORKSPACE/.venv3/bin/activate
+pip install https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.3.0/zeroc_ice-3.6.5-cp38-cp38-linux_x86_64.whl
+pip install -U pip future
+pip install markdown
+pip install reportlab # For figure
```

Comment:
Likely unrelated to the scope of this PR, this raises the question of whether this dependency should be managed via `omero-figure` setup.py.

Reply:
this is out of scope. We need to solve the problem of where the export to PDF is happening. It is currently a server-side script i.e. figure is a web app with a foot in the server

Reply:
Oh right. Ignore my comment, I misread and thought this was the Web configuration. Indeed the problem is harder. If the PDF generation start happening client-side, the dependencies can be managed with the app.